python/mlc_llm/serve/async_engine.py

```diff
@@ -145,6 +145,13 @@ def __init__(
             prefill_chunk_size,
             self.conv_template_name,
         ) = _process_model_args(models)
+
+        model_lib_idx_mult = len(model_args) // len(models)
+        for i, model in enumerate(models):
+            # model_args:
+            # [model_lib_path, model_path, device.device_type, device.device_id] * N
+            model.model_lib_path = model_args[i * model_lib_idx_mult]
```

Comment:
Could you elaborate a bit about the rationale of this change?

Reply:
Models compiled using JIT don't have model_lib_path in the model info class which is used later on for estimating max sequence length, hence we set model_lib_path to the path returned from _process_model_args function call.